pkg/policy/k8s/service.go

```diff
@@ -39,12 +39,13 @@ func (p *policyWatcher) onServiceEvent(event k8s.ServiceNotification) {
 // added, removed, its endpoints have changed, or its labels have changed).
 // This function then checks if any of the known CNP/CCNPs are affected by this
 // change, and recomputes them by calling resolveCiliumNetworkPolicyRefs.
-func (p *policyWatcher) updateToServicesPolicies(svcID k8s.ServiceID, newSVC, oldSVC *k8s.Service) error {
+func (p *policyWatcher) updateToServicesPolicies(svcID k8s.ServiceID, newSVC, oldSVC *k8s.MinimalService, newEps, oldEps *k8s.MinimalEndpoints) error {
 	var errs []error
+	endpointsChanged := !newEps.DeepEqual(oldEps)
```

Comment:
Nit. Could be worth moving this fix to a dedicated commit, so that is it easier to backport if necessary.

Reply:
Done.